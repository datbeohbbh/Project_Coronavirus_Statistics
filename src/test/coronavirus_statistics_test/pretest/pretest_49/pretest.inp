50 100
q 1 15 438
q 1 19 147
q 1 13 485
? 4 1
? 4 1
q 1 45 341
q 1 4 283
q 1 45 422
q 2 45 621
? 4 3
q 1 50 430
? 7
q 1 36 44
q 2 50 284
q 3 15 341
q 2 45 19
q 1 24 250
? 4 2
? 4 4
q 1 34 211
q 1 48 101
q 3 36 30
? 4 3
? 4 5
? 7
? 7
q 1 11 120
q 1 17 2
q 3 45 38
q 3 11 97
q 1 31 208
? 4 8
q 1 26 248
q 1 18 416
? 4 9
? 7
? 4 2
? 7
? 4 10
q 2 45 62
? 4 9
q 2 15 87
? 4 13
? 4 15
q 1 38 359
? 4 5
q 1 11 384
? 4 2
q 2 15 7
? 6
q 1 29 146
? 7
? 4 17
q 1 1 475
q 1 36 500
q 3 24 137
? 4 15
q 1 20 460
q 3 11 371
q 2 15 1
? 4 6
q 1 8 421
? 7
? 4 17
? 7
q 1 5 9
? 4 15
? 7
q 1 44 296
? 7
q 2 31 28
q 3 15 1
? 6
? 7
q 3 45 21
? 7
? 7
? 4 8
? 6
? 4 1
q 2 36 475
? 4 12
q 1 32 265
q 1 23 158
? 7
? 7
? 4 12
? 6
? 4 16
? 4 24
? 6
? 4 14
? 4 3
? 4 16
? 7
? 4 9
? 4 22
? 6
? 4 21
? 6