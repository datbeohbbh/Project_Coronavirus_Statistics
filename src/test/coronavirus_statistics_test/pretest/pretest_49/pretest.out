Query ? 4 1 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 147
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 147
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 283
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 97
-> Has recovered : 0
-> Was dead : 341

Query ? 4 4 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 146
-> Has recovered : 284
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 101
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 146
-> Has recovered : 284
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 341 people

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 341 people

Query ? 4 8 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 147
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 208
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 341 people

Query ? 4 2 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 14
-> Has recovered : 0
-> Was dead : 30

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 341 people

Query ? 4 10 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 211
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 208
-> Has recovered : 0
-> Was dead : 0

Query ? 4 13 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 283
-> Has recovered : 0
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 485
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 23
-> Has recovered : 0
-> Was dead : 97

Query ? 4 2 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 10
-> Has recovered : 87
-> Was dead : 341

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 341 people

Query ? 4 17 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 485
-> Has recovered : 0
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 416
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 113
-> Has recovered : 0
-> Was dead : 137

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 4 17 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 460
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 4 15 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 359
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 4 8 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 146
-> Has recovered : 284
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 4 1 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 1
-> Has recovered : 95
-> Was dead : 342

Query ? 4 12 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 180
-> Has recovered : 28
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 4 12 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 158
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 4 16 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 265
-> Has recovered : 0
-> Was dead : 0

Query ? 4 24 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 485
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 4 14 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 211
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 2
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 265
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 11
Quantity: 468 people

Query ? 4 9 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 146
-> Has recovered : 284
-> Was dead : 0

Query ? 4 22 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 460
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

Query ? 4 21 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 421
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 45
Quantity: 702 people

